src/web/app/pages-instructor/instructor-session-modal-page.component.ts

```diff
@@ -161,10 +164,11 @@ export abstract class InstructorSessionModalPageComponent extends InstructorSess
               isSelected: selectAllRespondents && !giverSet.has(instructor.email),
             } as InstructorListInfoTableRowModel));
 
-        modalRef.result.then((respondentsToRemind: any[]) => {
+        modalRef.result.then((reminderResponse: ReminderResponseModel) => {
           this.isSendReminderLoading = true;
           this.feedbackSessionsService.remindFeedbackSessionSubmissionForRespondents(courseId, feedbackSessionName, {
-            usersToRemind: respondentsToRemind.map((m: any) => m.email),
+            usersToRemind: reminderResponse.respondentsToSend.map((m: any) => m.email),
```

Comment:
Let's remove the any type here. Not much point using any in this case (as it doesn't really give us any additional information). We used to enforce having a type in typescript, but we do not do that anymore. It's ok to leave it out the type if it is obvious from context.

```suggestion
            usersToRemind: reminderResponse.respondentsToSend.map(m => m.email),
```

Reply:
I see, but would this change make it inconsistent compared to other similar methods, i.e. `remindFeedbackSessionSubmissionForRespondents()` in `sendReminderToStudents()` in `instructor-session-result-page.component.ts` and `remindResultsLinkToRespondents()` in `resendResultsLinkToRespondentsEventHandler()` in `instructor-session-modal-page.component.ts`?

Reply:
True, but it's also true that using `any` isn't really considered best practice. We have been slowly trying to enforce only adding types when useful or when not immediately obvious from context. I think it's worth the tradeoff here.